static/docker-compose.yml

```diff
@@ -118,12 +118,14 @@ services:
       LETS_ENCRYPT_EMAIL: "${LETS_ENCRYPT_EMAIL:-}"
       OVERRIDE_SENTRY_FRONTEND_ENABLED: "${OVERRIDE_SENTRY_FRONTEND_ENABLED-}"
       USE_CUSTOM_CERT: "${USE_CUSTOM_CERT-}"
+      WAF_ENABLED: "${WAF_ENABLED:-true}"
     ports:
     - 0.0.0.0:80:80/tcp
     - 0.0.0.0:443:443/tcp
     restart: always
     volumes:
     - letsencrypt:/etc/letsencrypt:rw
+    - ${PLEXTRAC_HOME:-.}/volumes/naxsi-waf/customer_curated.rules:/etc/nginx/conf.d/customer_curated.rules:r
```

Comment:
Given above, we can remove this unless its expected and required by the container.

Reply:
It is expected by the container. The idea is to populate this file with a comment and at mount the management util combines the pextrac curated rules and any customer curated custom rules into the same file